pkg/deploy-manager/config.go

```diff
@@ -27,8 +27,11 @@ const DefaultStorageClusterName = "test-storagecluster"
 // DefaultStorageClassRBD is the name of the ceph rbd storage class the test suite installs
 const DefaultStorageClassRBD = DefaultStorageClusterName + "-ceph-rbd"
 
-// MinOSDsCount represents the minimum number of OSDs required for this testsuite to run.
-const MinOSDsCount = 3
+const minOSDsCount = 3
+
+func (t *DeployManager) getMinOSDsCount() int {
```

Comment:
if it's always going to return a constant, what benefit does this method add?

Reply:
This is in preparation for arbiter. MinOSDCount for arbiter is 4. This function will return the right value as it also has access to the storagcluster configuration and can check if arbiter is enabled.

Reply:
ok. That makes the intent clearer.
Can we add a generic comment for that?